modules/ROOT/pages/install/get-started.adoc

```diff
@@ -171,13 +219,11 @@ NAME   TYPE     DATA   AGE
 csb    	Opaque       5      7m58s
 ----
 
-To access the couchbase cluster UI console: 
+To access the Couchbase Web Console:
 
 [source,console]
 ----
 $ kubectl port-forward couchbase-instance-winhhoku-0000 8091
```

Comment:
Where does thins pod name come from?  Thin air it looks like.  You'll need a way to map your service instance to the pod name or a user is going to complain.